src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss

```diff
@@ -0,0 +1,58 @@
+.headerLeft {
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
+
+    & > * {
+        margin-right: var(--spacing-small);
+    }
+
+    .headerTypography {
+        flex-grow: 1;
+        width: 250px;
+    }
+}
+
+.headerRight {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.headerChips {
+    & > * {
+        margin-left: var(--spacing-small);
+    }
+}
```

Comment:
The moonstone component already manage the spacing between status if you put directly the components inside the `status` props, or you can manage it at the `Headerbadges` component level.

Reply:
this class is actually useless